src/services/profiles/profiles.ts

```diff
@@ -32,11 +46,28 @@ export default (
   },
 
   async create(requestBody, options) {
-    return (await client.post(httpAuth, '/', requestBody, options)).data;
+    const customProperties = ['custom_fields', 'groups.custom_fields'];
```

Comment:
Would it make sense to add just a single test for each of these methods to prove they're working? (No need to add all the test you added for `find` to all these other methods though)

Reply:
Gave it go but with the current state of the tests it would be a large and even more annoying change, will add this as a question to the questions document to see if we want to do this for next PR's and maybe make a new one for the profiles.